frontend/src/components/Blocker.tsx

```diff
@@ -1,60 +1,158 @@
 import Modal from "antd/lib/modal/Modal";
-import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useEffect,
+  useState,
+} from "react";
+import { Range } from "react-range";
+import styled from "styled-components";
+import star from "../assets/star.svg";
+import background from "../assets/slider.svg";
 
 interface BlockerProps {
-    children: ReactElement,
-    afterSeconds: number,
-    message: string,
-    repeatable: boolean
+  children: ReactElement;
+  afterSeconds: number;
+  repeatable: boolean;
```

Comment:
is there a reason why we need `repeatable`? it's always `false` for now.

Reply:
That is a good point. I believe at one point there was a discussion of whether they should be repeated, and don't remember the ending to that conversation. I am going to leave it now and we can ask the researchers if that is something they would want at the next client meeting.